getting_started/5_Quantum_Teleportation.ipynb

```diff
@@ -0,0 +1,285 @@
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from braket.circuits import Circuit\n",
+    "from braket.devices import LocalSimulator\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import time\n",
+    "%matplotlib inline\n",
+    "\n",
+    "device = LocalSimulator()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "T  : |0|1|2|3|4|5|6|\n",
+      "                    \n",
+      "q0 : -H-T-Z-H-C-H-C-\n",
+      "              |   | \n",
+      "q1 : -H-C-----X-C-|-\n",
+      "        |       | | \n",
+      "q2 : ---X-------X-Z-\n",
+      "\n",
+      "T  : |0|1|2|3|4|5|6|\n"
+     ]
+    }
+   ],
+   "source": [
+    "qt = Circuit()\n",
+    "\n",
+    "# Prepare Initial State ( you can experiment with different states)\n",
+    "\n",
+    "#qt.i(0)            # Initial State is 0\n",
+    "#qt.x(0)             # Initial State is 1\n",
+    "#qt.h(0)             # Initial State is a 50:50 superposition\n",
+    "#qt.h(0).t(0).h(0)     # Initial State is 85:15 superposition\n",
+    "qt.h(0).t(0).z(0).h(0) # Initial State is 15:85 superposition\n",
+    "#qt.ry(0,2*np.pi/5)      # Initial State is 66:33 superposition\n",
+    "######\n",
+    "\n",
+    "\n",
+    "\n",
+    "# Prepare a Bell state between qubits 1 &2. Qubit 0 & 1 is with Alice. Qubit 2 is with Bob\n",
+    "qt.h(1).cnot(1,2)\n",
+    "\n",
+    "# Add CNOT and Hadamard to Alice's part of the circuit\n",
+    "qt.cnot(0,1)\n",
+    "qt.h(0)\n",
+    "\n",
+    "\n",
+    "# Add Conditional gates to Bob's Qubit\n",
+    "qt.cnot(1,2)\n",
+    "qt.cz(0,2)\n",
+    "print(qt)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Now run the circuit"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "task = device.run(qt,shots=1000)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Measurement_counts of all 3 qubits: Counter({'111': 223, '001': 219, '101': 213, '011': 206, '100': 40, '010': 36, '110': 32, '000': 31})\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "result = task.result()\n",
+    "print(f\"Measurement_counts of all 3 qubits: {result.measurement_counts}\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To simulate partial mesurement of just Bob's qubit (qubit-2), we'll extract just the qubit-2 measurement counts and plot the results."
```

Comment:
"To simulate partial mesurement of" suggested change: "To obtain the measurement results of "

Reply:
Updated